docs/get_started/getting_started_python.md

```diff
@@ -61,22 +61,19 @@ $ python -m pip install numpy absl-py
 
 ## Building
 
-From the *parent* directory of the IREE git repository clone, create and enter a
-build directory, such as:
```

Comment:
This differs a bit from the instructions below


> We continue to assume that we are in the build directory where we made the build
in the previous section.

In particular, `ctest` and the other commands must be run from the build directory.

Maybe replace the "We continue to..." line with "Move into the build directory"? Then the build commands would be the same as the other pages, and the rest of the document can continue to work from the build directory.

Reply:
Thanks, I didn't notice. I will just update according to your suggestion.